devel/libsdl2/Portfile

```diff
@@ -32,9 +33,10 @@ platform macosx {
         set lion_sdkpath ${developer_dir}/SDKs/MacOSX10.7.sdk
         if {[file exists $lion_sdkpath]} {
             configure.sdkroot $lion_sdkpath
+            compiler.blacklist *gcc* { clang < 211 }
         } else {
             pre-fetch {
-                error "Building $name requires OS X 10.7 SDK or later"
+                error "Building $name on MacOSX 10.6 requires the MacOSX10.7.sdk to be present in ${developer_dir}/SDKs/"
```

Comment:
Sorry to nitpick this further, but the OS name is "Mac OS X 10.6" not "MacOSX 10.6".

I've also been in the habit of including the port version number in messages of this kind, e.g. `Building ${name} @${version} on Mac OS X 10.6...`